src/main/java/teammates/common/util/Url.java

```diff
@@ -1,15 +1,18 @@
 package teammates.common.util;
 
+/**
+ * A simple class to encapsulate URLs.
```

Comment:
By convention, the first line goes something like this:
`The Url class represents a URL string.`
Then, we can go on to say,
`It provides methods to manipulate the URL string and extract values from it.`
Notes:
* No mention that the class is 'simple' (it doesn't add any value), or the internal storage mechanism (not relevant to the reader - header comment are used to generate javadoc read by the users of this class, not maintainers). 
* The description of the methods is sufficiently general so that we don't have to change it every time we add methods.